taskcluster/tc_staging.rst

```diff
@@ -129,6 +129,24 @@ Once this goes green, the logs will link you to the decision task ID.
 
 Watch the task group. Ideally whatever you're concerned about (in this case it was docker-worker artifact uploads) will go green.
 
+Test the Github Integration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A `Github app`_ is connected to the Taskcluster staging instance and installed
+on the `stage-taskcluster-test`_ repository. Run some tasks by creating a pull
+request, verify that they are scheduled and show up in the Github checks
+interface. Merge the pull request to create a `github-push` event and verify the
+task is scheduled again.
+
+.. note::
+
+   This repo is not yet configured via `ci-configuration`, so you'll need to add scopes manually
+   by `creating the appropriate roles in the web ui`_.
```

Comment:
I was unsure whether we wanted to have this repo in the regular `ci-configuration` given it's not enabled with the `firefox-ci` instance. I figured creating roles manually for now wasn't too hard and we could add it later if it's too much of a pain.

Reply:
We have staging-only configs in ci-config already, so it's probably the right place.

Reply:
Thanks, I'll land this for now and delete this note sans review once I have the scopes set up properly. I'm a bit unsure what pools to use etc, so not sure when I'll get around to it.